objcgen/driver.cs

```diff
@@ -53,20 +55,35 @@ public static int Main2 (string [] args)
 				Console.WriteLine ($"Embeddinator-4000 v0.1 ({Info.Branch}: {Info.Hash})");
 				Console.WriteLine ("Generates target language bindings for interop with managed code.");
 				Console.WriteLine ("");
-				Console.WriteLine ($"Usage: {Path.GetFileName (System.Reflection.Assembly.GetExecutingAssembly ().Location)} [options]+ ManagedAssembly1.dll ManagedAssembly2.dll");
+				Console.WriteLine ($"Usage: {Path.GetFileName (System.Reflection.Assembly.GetExecutingAssembly ().Location)} [options]+ ManagedAssembly1.dll [ManagedAssembly2.dll ...]");
 				Console.WriteLine ();
 				os.WriteOptionDescriptions (Console.Out);
 				return 0;
 			case Action.Version:
 				Console.WriteLine ($"Embeddinator-4000 v0.1 ({Info.Branch}: {Info.Hash})");
 				return 0;
 			case Action.Generate:
-				return Generate (assemblies, shared);
+				try {
+					return Generate (assemblies, shared);
+				} catch (NotImplementedException e) {
+					throw new EmbeddinatorException (1000, $"The feature `{e.Message}` is not currently supported by the tool");
+				}
 			default:
 				throw ErrorHelper.CreateError (99, "Internal error: invalid action {0}. Please file a bug report with a test case (https://github.com/mono/Embeddinator-4000/issues)", action);
 			}
 		}
 
+		static string outputDirectory = ".";
+
+		public static string OutputDirectory {
+			get { return outputDirectory; }
+			set {
+				if (!Directory.Exists (value))
```

Comment:
It's not typo friendly (e.g. "../../../test/bin/debug" that has an extra, or missing, "../") and they are more common from a (currently) CLI tool. OTOH we can warn when we create.